tensorflow/stream_executor/blas.h

```diff
@@ -101,13 +101,72 @@ enum class ComputationType {
   kI32,         // 32-bit integer
   kComplexF32,  // Complex number comprised of two f32s.
   kComplexF64,  // Complex number comprised of two f64s.
+  // The below values are only supported for BlasLt routines (both real and
+  // complex).
+  kF32FastTF32, // 32-bit floating-point with reduced (>=10-bit) mantissa
+  kF32FastBF16, // 32-bit floating-point with reduced (7-bit) mantissa
 };
 
 // Converts a ComputationType to a string.
 std::string ComputationTypeString(ComputationType ty);
 
 std::ostream &operator<<(std::ostream &os, ComputationType ty);
 
+// Type with which inputs and outputs of a blaslt routine are performed.
+enum class DataType {
```

Comment:
dnn.proto also has DataType. Maybe pull it out of the dnn namespace and reuse it?

Reply:
Done (I just did it by including dnn.h; let me know if this is undesirable).